src/Event/ValueObjects/Status.php

```diff
@@ -15,7 +15,7 @@ class Status
      */
     private $value;
 
-    public function __construct(string $value)
+    private function __construct(string $value)
```

Comment:
I'm pretty sure you're going to need the validation later in a `::fromNative()` call though 😅